gcloud/storage/blob.py

```diff
@@ -109,6 +111,23 @@ def path_helper(bucket_path, blob_name):
         """
         return bucket_path + '/o/' + quote(blob_name, safe='')
 
+    @staticmethod
```

Comment:
Just make this a function in the module instead of a `staticmethod`? Unless there is a reason not to?